test/dotnet-test.Tests/GivenDotnetTestBuildsAndRunsTestsWithDifferentOptions.cs

```diff
@@ -258,6 +258,88 @@ public void RunTestProjectSolutionWithArchOption_ShouldReturnZeroAsExitCode(stri
             result.ExitCode.Should().Be(ExitCodes.Success);
         }
 
+        [InlineData(TestingConstants.Debug)]
+        [InlineData(TestingConstants.Release)]
+        [Theory]
+        public void RunTestProjectSolutionWithRuntimeOption_ShouldReturnZeroAsExitCode(string configuration)
+        {
+            TestAsset testInstance = _testAssetsManager.CopyTestAsset("TestProjectWithTests", Guid.NewGuid().ToString()).WithSource();
+
+            string runtime = CommonOptions.ResolveRidShorthandOptionsToRuntimeIdentifier(string.Empty, "x64");
+
+            CommandResult result = new DotnetTestCommand(Log, disableNewOutput: false)
+                                    .WithWorkingDirectory(testInstance.Path)
+                                    .WithEnableTestingPlatform()
+                                    .Execute(CommonOptions.RuntimeOption.Name, runtime,
+                                             TestingPlatformOptions.ConfigurationOption.Name, configuration);
+
+            Assert.Matches(RegexPatternHelper.GenerateProjectRegexPattern("TestProject", TestingConstants.Passed, true, configuration, runtime: runtime), result.StdOut);
+
+            result.ExitCode.Should().Be(ExitCodes.Success);
```

Comment:
Can we combine multiple exit code? If not then the enum/class should be called `ExitCode` and not `ExitCodes`.